influxdata/tokenizer.go

```diff
@@ -316,26 +331,56 @@ func (t *Tokenizer) NextFieldBytes() (key []byte, kind ValueKind, value []byte,
 		return nil, Unknown, nil, t.syntaxErrorf("unexpected character %q after field", nextc)
 	}
 	t.take(fieldSeparatorSpace)
-	if !t.ensure(1) || t.at(0) == '\n' {
+	if t.takeEOL() {
 		t.section = endSection
 		return fieldKey, fieldKind, fieldVal, nil
 	}
 	t.section = TimeSection
 	return fieldKey, fieldKind, fieldVal, nil
 }
 
+// takeEOL consumes input up until the next end of line.
```

Comment:
Does this consume input `up until the next end of line`?  It looks like it just consumes a single `\n` or `\r\n` if the tokenizer is at that character?

Reply:
good point.

Reply:
fixed